src/ray/util/util.cc

```diff
@@ -35,11 +35,12 @@
 #include "ray/util/filesystem.h"
 #include "ray/util/logging.h"
 
+namespace {
 /// Uses sscanf() to read a token matching from the string, advancing the iterator.
 /// \param c_str A string iterator that is dereferenceable. (i.e.: c_str < string::end())
 /// \param format The pattern. It must not produce any output. (e.g., use %*d, not %d.)
 /// \return The scanned prefix of the string, if any.
-static std::string ScanToken(std::string::const_iterator &c_str, std::string format) {
```

Comment:
Reading through the codebase, using anonymous namespace seems to be the fashion, just to keep coding consistency.

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/misc/use-anonymous-namespace.html also preferred 👍🏽